tests/java/parsing/EnumEllipsis.java

```diff
@@ -0,0 +1,2 @@
+enum $A { ... }
```

Comment:
this is not valid Java code, it should not be in tests/java/parsing/

Reply:
Is there a good place to put in Pfff? Otherwise I can remove the test and add the test in Semgrep instead.

Reply:
Yep I did that. For some languages I had a pfff/tests/<lang>/sgrep/ subdirectory to put those tests, but now it's better to have them in semgrep I think.